js/branches-info-wl.js

```diff
@@ -1,37 +1,100 @@
 var IntWL = {
-  cn: {
-    name: "中文",
-    head: "其他语言",
-    url: "https://scp-wiki-cn.wikidot.com/",
-    id: "530812",
-    category: "wanderers",
-  },
   en: {
     name: "English",
     head: "Languages",
-    url: "https://wanderers-library.wikidot.com/",
+    url: "http://wanderers-library.wikidot.com/",
```

Comment:
Wikidot supports these URLs, redirecting them to the HTTP version. I've generally preferred the HTTPS links because if/when it switches then it'll make the transition easier if some links are already updated.

Reply:
Sounds good.

Worth noting that that's not always the case - the ajax module connector endpoint for example will reject a HTTPS connection if the site is set up only to use HTTP (and vice versa, if the site is set up *only* to use HTTPS) - but the only module in use for the Interwiki currently is `quickmodule.php` which doesn't seem to care.

Reply:
Oh jeez, is that the case? Wikidot why